### ISSUE TITLE
Surface Pro Dock Requirement is Misleading

### ISSUE THREAD

**Issue by franklesniak** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4211]
Hello, I read the following text:
"One of the following docking station options to secure a tablet to the meeting room table."
And because I'm spinning up a room system at home, I thought: hey, I don't need to secure the tablet to the meeting room table because I don't have a meeting room table. Therefore, I did not purchase a docking station for the Surface Pro.

Fast forward a few days, and I discovered that I cannot complete the room setup wizard because I have no dock.

The text is misleading. Instead of "One of the following docking station options to secure a tablet to the meeting room table.", I recommend a simple and clear, "Surface Pro devices require one of the following docking station options."

Alternatively, it would be great to remove the dock requirement from the software - or at least allow a bypass as having a dock seems unnecessary for testing purposes.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5bf01435-a73b-4bf2-5845-e4cf50c4c3a2
* Version Independent ID: 166e1800-80b1-2496-7160-3040c5b23c3d
* Content: [Microsoft Teams Rooms requirements - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/rooms/requirements)
* Content Source: [Teams/rooms/requirements.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/requirements.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@franklesniak  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by dariomws** [comment]
Hi @franklesniak, thank you for your feeback.

I'm applying this update to the article.

About the feature request please go to:
https://microsoftteams.uservoice.com/

**Comment by franklesniak** [comment]
For anyone else looking for my suggestion on UserVoice, I found someone else had already posted it. Here it is: https://microsoftteams.uservoice.com/forums/555103-public/suggestions/38450635-teams-room-system-development-kit

**Comment by scanum** [comment]
@franklesniak Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by franklesniak** [comment]
Hey all, thanks for taking a crack at this. I don't feel it's been addressed, though. The page now reads, "Surface Pro devices require one of the following docking station options **to secure a tablet to the meeting room table**."

I think this is still unclear. Surface Pro devices require a docking station **because the software insists upon it**; it's not just because it secures the Surface Pro to the meeting room table. This confusion is what led to my misunderstanding in the first place. I did not want to secure the tablet to the table, so I thought I did not need a dock.

I think it would be more clear if instead of, "Surface Pro devices require one of the following docking station options to secure a tablet to the meeting room table.", it read:
**Surface Pro devices require one of the following docking stations.**
